### DISCUSSION TITLE
[Issue forms] Add a label depending on what item is selected in a dropdown menu

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#4958] [Projects and Issues] [answered]
I'll be using [this issue form](https://github.com/makedeb/makedeb/issues/new?assignees=&labels=bug&template=bug-report.yml) as an example for my request.

Under `Installation source`, three options are available:

- AUR
- MPR
- APT Repository

I'd like the ability for the `AUR` label in my repository to automatically be added when `AUR` is selected, and for it to do likewise with the `MPR` tag when `MPR` is selected.

**Comment by lukehefson** [comment, marked as answer]
Hey @hwittenborn! I _think_ your request is a dupe of https://github.com/github/feedback/discussions/4266. Feel free to add a <kbd>👍</kbd>, upvote and/or comment there if you have additional feedback.
